spec/models/rating/averager_data_spec.rb

```diff
@@ -13,7 +13,14 @@
     end
 
     it 'returns the average of number of records for the given resource type' do
-      expect(result.as_json['count_avg'].to_s).to eq '1.333333333333333333'
+      case ENV.fetch('DB')
+      when 'mysql'
+        expect(result.count_avg).to eq(BigDecimal('1.333333333333333333'))
```

Comment:
@wbotelhos For precision matches you can either:

1. Check against a rounded value
2. Use a be_within matcher

I don't think this structure is desirable - it doesn't make manifest that this is a precision issue.  Using one of the two above, along with a comment, seems like it would be more clear.

Reply:
Yeah, do you preferer we round it? I think it's a good idea.

Reply:
Generally speaking lots of decimal places of precision are a code smell - they are more likely to be an artifact of how the calculation is done, rather than reflecting any actionable truth.  That's especially true in an application like this - there's more uncertainty introduced by the limited sample size than is implied by the returned precision.

For a rating system like this one, does it make any sense to have more than a few decimal places of precision (2 or 3?).  If so, what is that that additional precision communicating?

So you may want to consider rounding in the library, rather than in the specs.

Reply:
I asked Evan Miller if the truncation could invalidate his theory, but I didn't receive an answer. Since the PR is good to go, let's merge it and open another issue about permanently truncating the results to two decimals number.